airbyte-server/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     implementation project(":airbyte-json-validation")
     implementation project(':airbyte-protocol:models')
     implementation project(':airbyte-scheduler')
+    implementation project(':airbyte-workers')
```

Comment:
:( having server to pull the scheduler was already problematic and now we pull in workers.

Reply:
You should really question the library structure here. Should we have a temporal library instead?

Reply:
And maybe what we need is a scheduler client library so we can stop depending on scheduler

Reply:
All good!! Just want to make sure it was a conscious decision!

Reply:
To keep myself honest add an [issue](https://github.com/airbytehq/airbyte/issues/2398) to track that we do this.